src/dolphin/install/macos.ts

```diff
@@ -7,17 +7,19 @@ import { extractDmg } from "./extractDmg";
 export async function installDolphinOnMac({
   assetPath,
   destinationFolder,
+  shouldBackupUserFolder,
   log = console.log,
 }: {
   assetPath: string;
   destinationFolder: string;
+  shouldBackupUserFolder: boolean;
   log?: (message: string) => void;
 }) {
   const backupLocation = destinationFolder + "_old";
   const dolphinResourcesPath = path.join(destinationFolder, "Slippi Dolphin.app", "Contents", "Resources");
 
-  const alreadyInstalled = await fs.pathExists(dolphinResourcesPath);
-  if (alreadyInstalled) {
+  const bkpUserFolder = shouldBackupUserFolder && (await fs.pathExists(dolphinResourcesPath));
+  if (bkpUserFolder) {
```

Comment:
```suggestion
  const backupUserFolder = shouldBackupUserFolder && (await fs.pathExists(dolphinResourcesPath));
  if (backupUserFolder) {
```
nit: can we just call this backup for consistency?